tests/full-disk-encryption.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { test, expect } from '@playwright/test';
+import { StoragePage } from './pages/storage-page';
+import { EncryptionPasswordPopup, encryptionPasswordPopup } from './pages/encryption-password-popup';
+import { mainPagePath } from "../lib/installer";
+
+test.describe('The main page', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto(mainPagePath());
+  });
+
+  test('Full-disk-encryption', async ({ page }) => {
+    await page.getByRole('link', { name: 'Storage' }).click();
+
+    const storagePage = new StoragePage(page);
+    await storagePage.useEncryption();
+
+    const passwordPopup = new EncryptionPasswordPopup(page);
+    await passwordPopup.fillPassword('nots3cr3t');
+    await passwordPopup.fillPasswordConfirm('nots3cr3t');
+    await passwordPopup.accept();
+
+    await storagePage.validateEncryptionIsUsed();
+    await storagePage.backToMainPage();
```

Comment:
just `storagePage.back()` all the `back` button should go to main page, because the Agama design, sort of single-page app with popups. I downvote my own suggestion! :) but up to you.